planning/scenario_planning/common/motion_velocity_optimizer/scripts/closest_velocity_checker.py

```diff
@@ -132,7 +132,7 @@ def timerCallback(self):
         self.printInfo()
 
     def CallBackAwEngage(self, msg):
-        self.autoware_engage = msg
+        self.autoware_engage = msg.data
```

Comment:
I think the message type of engage was modified in ros2 porting.

Reply:
fixed.